### ISSUE TITLE
[MU4 Issue] Virtual Keyboard can't remember its zoom setting

### ISSUE THREAD

**Issue by oMrSmith** [musescore/MuseScore#16025]
If you zoom out the virtual keyboard
by using CTR + Scroll
and then toggle the virtual keyboard off and on again,
the zoom-setting is reset.

**Comment by rgreen5** [comment]
Identical to  #13883.

**Comment by cbjeukendrup** [comment]
No, this issue is about the zoom of the keyboard itself, while your issue is about the size of its containing panel.
